package.json

```diff
@@ -162,6 +162,7 @@
     "mocha-junit-reporter": "^2.2.1",
     "nock": "^13.5.4",
     "nodemon": "^3.1.4",
+    "openapi-typescript": "^7.0.3",
```

Comment:
Hopefully pinning version here will stop us having conflicting versions of the API definitions!